crux/src/Crux/Config/Common.hs

```diff
@@ -477,6 +490,16 @@ cruxOptions = Config
         "The file to store the interaction with the online goal solver (if any) (default: none)"
         $ ReqArg "FILE" $ \f opts -> Right opts { onlineSolverOutput = Just f }
 
+      , Option [] ["offline-solver-output"]
+        (unlines
+          [ "A template to use for files to store interactions with offline goal solvers"
+          , "(if any) (default: none). For example, if the template is `offline-output.smt2`,"
+          , "then each file will be named `offline-output-<goal number>-<solver name>.smt2,"
+          , "where <goal number> is the number of the goal that was proven (starting at 0)"
+          , "and <solver name> is the name of the solver used to dispatch that goal."
```

Comment:
This text duplication makes me sad.

Reply:
Yes, a good point. Luckily, these strings are only used within this one module, so it was straightforward to de-duplicate it. (I suppose we should de-duplicate the _other_ strings in this module, but I'm not going to do that here.)